### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by alifarhid**

### CODE REVIEW

File: notebooks/mortgage.ipynb [hunk 1]
```diff
@@ -179,7 +219,7 @@
     "        \"mortgage_insurance_type\": np.float64,\n",
     "        \"relocation_mortgage_indicator\": pd.CategoricalDtype([\"N\", \"Y\"]),\n",
     "    }\n",
-    "    acq_file = \"mortgage/acq/Acquisition_\" + str(year) + \"Q\" + str(quarter) + \".txt\"\n",
+    "    acq_file = \"/shared/hadia/mortgage_data/acq/Acquisition_\" + str(year) + \"Q\" + str(quarter) + \".txt\"\n",
```
Comment (hadia206) [inline notebooks/mortgage.ipynb:222]:
```suggestion
    "    acq_file = \"mortgage_data/acq/Acquisition_\" + str(year) + \"Q\" + str(quarter) + \".txt\"\n",
```

File: notebooks/mortgage.ipynb [hunk 2]
```diff
@@ -338,15 +405,24 @@
...
+   },
    "outputs": [],
    "source": [
     "%%px\n",
     "@bodo.jit(distributed=[\"final_gdf\"], cache=True)\n",
     "def morg_func(year, quarter, perf_file):\n",
     "    t1 = time.time()\n",
     "    # read names file\n",
-    "    names_file = \"mortgage/names.csv\"\n",
+    "    names_file = \"/shared/hadia/mortgage_data/names.csv\"\n",
```
Comment (hadia206) [inline notebooks/mortgage.ipynb:425]:
```suggestion
    "    names_file = \"mortgage_data/names.csv\"\n",
```

**Review by hadia206 — APPROVED** [review]
thanks @alifarhid

**Comment by alifarhid** [comment]
Thanks to you indeed @hadia206

**Pull request merged**